lambda_src/lambda_function.py

```diff
@@ -155,26 +155,29 @@ def sync_flow(event: Any, context: Any = None) -> Dict[Text, Any]:
             'statusCode': 200,
             'body': b64encode(compress(data_dumps.encode())).decode(),
             'isBase64Encoded': True,
-            'headers': {'Content-Encoding': 'gzip'}
+            'headers': {'Content-Encoding': 'gzip'},
         }
 
-    if len(response) > 6_000_000:
-        response = dumps(
-            {
-                'data': [
-                    [
-                        rn,
-                        {
-                            'error': (
-                                f'Response size ({len(response)} bytes) will likely'
-                                'exceeded maximum allowed payload size (6291556 bytes).'
-                            )
-                        },
+    if len(dumps(response)) > 6_000_000:
```

Comment:
per [this blog post](https://zaccharles.medium.com/deep-dive-lambdas-response-payload-size-limit-8aedba9530ed), I think we can change the constant to 6_291_556

Reply:
Makes sense 👍